src/main/java/gameframework/motion/MoveStrategyKeyboard.java

```diff
@@ -25,8 +25,11 @@ public MoveStrategyKeyboard(SpeedVector speedVector) {
 	}
 
 	/**
-	 * @param alwaysMove is a boolean value that decide if a player moves continually or not. (True by default)
-	 * @param speedVector is a given custom speedVector for the strategy.
+	 * @param alwaysMove
+	 *            is a boolean value that decide if a player moves continually
+	 *            or not. (True by default)
+	 * @param speedVector
+	 *            is a given custom speedVector for the strategy.
```

Comment:
please make sure your pull request only contains changes that are related. This one looks unrelated.